assets/src/domain/eventEditor/ui/ticketAssignmentsManager/utils/ticketsWithNewQuantity.test.ts

```diff
@@ -0,0 +1,146 @@
+import { prepareEntitiesForUpdate, ticketsWithNewQuantity } from './';
+import { nodes as allDates } from '@edtrServices/apollo/queries/datetimes/test/data';
+import { nodes as allTickets } from '@edtrServices/apollo/queries/tickets/test/data';
+import { relationalData as existingData } from '@edtrServices/context/TestContext/data';
+import { assocPath, pathOr } from 'ramda';
+
+describe('TAM:ticketsWithNewQuantity', () => {
+	it('returns an empty object when existing relational data is empty', () => {
+		const ticketsToUpdate = prepareEntitiesForUpdate({
+			entity: 'tickets',
+			existingData: {},
+			newData: {},
+			relation: 'datetimes',
+		});
+
+		const ticketsWithChangedQuantity = ticketsWithNewQuantity({
+			allDates,
+			allTickets,
+			existingData,
+			ticketsToUpdate,
+		});
+
+		expect(ticketsWithChangedQuantity).toEqual({});
+	});
+
+	it('returns an empty object when new relational data is empty', () => {
+		const ticketsToUpdate = prepareEntitiesForUpdate({
+			entity: 'tickets',
+			existingData,
+			newData: {},
+			relation: 'datetimes',
+		});
+
+		const ticketsWithChangedQuantity = ticketsWithNewQuantity({
+			allDates,
+			allTickets,
+			existingData,
```

Comment:
if this is new relational data, should `existingData` be passed?

Reply:
i think that's how it calculates that there are new changes... by comparing against existing

Reply:
makes sense, I dunno why confused `ticketsWithNewQuantity` with a hook and thought that is should be aware of the `existingData` :facepalm:

Reply:
Yeah the comparison is with new changes